roles/openshift-applier/tasks/process-one-entry.yml

```diff
@@ -21,10 +22,16 @@
 
 - name: "Fail if template but no params is specified"
   fail:
-    msg: "Template specified, but no params file supplied"
+    msg: "Template specified, but no params file or variables supplied"
   when:
   - template|trim != ''
-  - params|trim == ''
+  - params|trim == '' or params_from_vars|trim == ''
+
+- name: "Create params file if there are params from vars"
+  include_tasks: create-one-param-from-vars.yml
+  when:
+  - params|trim != ''
```

Comment:
Why is required parameters file to exist here? Maybe I'm wrong but the intention is not to create this parameters file from provided parameters from inventory vars?